WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressValidationViewModel.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+import Yosemite
+
+final class ShippingLabelAddressValidationViewModel: NSObject {
+
+    typealias Section = ShippingLabelAddressValidationViewController.Section
+    typealias Row = ShippingLabelAddressValidationViewController.Row
+
+    let addressVerification: ShippingLabelAddressVerification
```

Comment:
nit: similar to the previous comment, maybe the view model can just provide `type` and `address` separately, since `ShippingLabelAddressVerification` is an implementation detail for the API validation call?